### PR TITLE
New Frontend Resource - CSS In JS

### PR DESCRIPTION
**Pull request by itsaakashpatel**
## Changes proposed

- Added a new Frontend Subcategory: Css In Js libraries
- Added emotion and styled-components.

## Screenshots

Xnip2023-05-21_23-13-44

### CODE REVIEW

**Review by CBID2 — CHANGES_REQUESTED** [review]
You’re close @itsaakashpatel!!😊 Just fix the typo I mentioned in the inline comment and you’re good to go! 😊

**Review by rupali-codes — CHANGES_REQUESTED** [review]
Hey, please add this in _ui generator_ instead creating a new subcategory

File: database/frontend/css-in-js.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+[
+    {
+        "name": "Emotion",
+        "description": "Emotion is a library designed for writing css styles with JavaScript. It provides powerful and predictable style composition.",
```
[the remark this answers is not in the document]
Comment (itsaakashpatel) [reply database/frontend/css-in-js.json]: Corrected! Thank you!

**Comment by itsaakashpatel** [comment]
@rupali-codes I added the required changes and rebased the PR with the `main` branch. Thanks!

**Review by CBID2 — APPROVED** [review]
Great job @Oyebolade! 😄 LGTM! 😄

**Review by rupali-codes — APPROVED** [review]
looks perfect

**Pull request merged**